contracts/vaults/balancer/BalancerVaultTypes.sol

```diff
@@ -75,6 +76,13 @@ struct BoostedOracleContext {
     UnderlyingPoolContext[] underlyingPools;
 }
 
+struct ComposableOracleContext {
+    /// @notice Amplification parameter
+    uint256 ampParam;
+    /// @notice Virtual supply
+    uint256 virtualSupply;
+}
+
 struct AuraStakingContext {
```

Comment:
@weitianjie2000 Can you remove the unused structs from this file?

Reply:
removed